components/raftstore/src/store/fsm/apply.rs

```diff
@@ -5109,6 +5161,18 @@ mod tests {
                     true
                 }
                 AdminCmdType::BatchSplit => true,
+                AdminCmdType::PrepareFlashback => {
+                    if let Some(region) = region_state.modified_region.as_ref() {
+                        return region.get_is_in_flashback();
+                    }
+                    false
+                }
+                AdminCmdType::FinishFlashback => {
+                    if let Some(region) = region_state.modified_region.as_ref() {
+                        return !region.get_is_in_flashback();
+                    }
+                    false
+                }
```

Comment:
We don't need to check the actual `is_in_flashback` flag here, as long as this cmd is a `PrepareFlashback` or `FinishFlashback`, the `should_write` always needs to be true to flush the change.